src/core/prompts/sections/custom-instructions.ts

```diff
@@ -297,3 +301,14 @@ The following additional instructions are provided by the user, and should be fo
 ${joinedSections}`
 		: ""
 }
+
+/**
+ * Check if a file should be included in rule compilation.
+ * Uses an allowlist approach to only include known rule file types,
```

Comment:
Hmm, we don't indicate anywhere in the UI what the known rule file types are though. I think there might be a variety of file extensions (including no extensions).

Reply:
I know it's not perfect, but I think I liked the blocklist approach better